web/docs/setup/deploy/README.md

```diff
@@ -6,3 +6,6 @@ it up in a runtime.
 We currently support the following runtimes:
 1. [AWS](deploy/aws)
 2. [Docker](deploy/docker)
+
+We also provide an Ansible role for bare-metal and traditional virtual machines
+in the [ansible directory](/ansible).
```

Comment:
We should add a documentation entry point and add `3. [Ansible](deploy/ansible)` to make it more consistent with the other deployment options. Maybe we can embed the existing ansible `README.md`?

Reply:
I like the idea of `3. [Ansible](deploy/ansible)`. I think this should hold the meat of the documentation, the documentation in the README only pointing to there.

Reply:
done.